fontir/src/ir.rs

```diff
@@ -189,7 +189,28 @@ impl GlyphPathBuilder {
     }
 
     pub fn qcurve_to(&mut self, p: impl Into<Point>) -> Result<(), PathConversionError> {
-        todo!("Support qcurve to {}", p.into());
+        // https://github.com/googlefonts/fontmake-rs/issues/110
+        // Guard clauses: degenerate cases
+        if self.is_empty() {
+            return self.move_to(p);
+        }
+        if self.offcurve.is_empty() {
+            return self.line_to(p);
+        }
+
+        // Insert an implied oncurve between every pair of offcurve points
+        for i in 0..(self.offcurve.len() - 1) {
+            // current offcurve to halfway to the next one
+            let lhs = self.offcurve[i];
+            let rhs = self.offcurve[i + 1];
```

Comment:
The [`windows`](https://doc.rust-lang.org/std/primitive.slice.html#method.windows) method may be appropriate here. 

Patiently waiting for [`array_windows`](https://doc.rust-lang.org/std/primitive.slice.html#method.array_windows) to land in stable which would allow
````rust
for [lhs, rhs] in self.offcurve.array_windows() {
````

Reply:
TIL, tha t seems handy indeed!